packages/core/src/providers/nodemailer.ts

```diff
@@ -51,32 +51,35 @@ export type NodemailerUserConfig = Omit<
 export default function Nodemailer(
   config: NodemailerUserConfig
 ): NodemailerConfig {
-  if (!config.server)
-    throw new AuthError("Nodemailer requires a `server` configuration")
+  if (!config.sendVerificationRequest && !config.server)
+    throw new AuthError(
+      "Nodemailer requires a `server` configuration when sendVerificationRequest isn't overridden"
+    )
 
   return {
     id: "nodemailer",
     type: "email",
     name: "Nodemailer",
-    server: { host: "localhost", port: 25, auth: { user: "", pass: "" } },
```

Comment:
i couldn't figure out why this server object was needed outside of development purposes